src/main.rs

```diff
@@ -213,7 +213,10 @@ pub mod content {
             word: &str,
             intermediate: &str,
         ) -> String {
-            dissimilar::diff(from, to)
+            dissimilar::diff(
+                &from.replace("┃🗌", "┃"), 
+                to
+            )
```

Comment:
This is a fix so that intermediate screen generation still works with the blanks I added.